test/razorpay/test_order.rb

```diff
@@ -66,6 +66,27 @@ def test_fetch_order_transfers
     assert_equal @order_id, order.id, 'order IDs do not match'
     refute_empty order.transfers["items"]
     assert_equal @transfer_id, order.transfers["items"][0]["id"]
+   end
+
+   def test_view_rto
+    stub_post("#{BASE_URI}/v1/orders/#{@order_id}/rto_review", 'fake_rto', {})
+    order = Razorpay::Order.fetch(@order_id).view_rto
+    assert !order.rto_reasons.empty?, 'orders should be more than one'
+   end
+
+   def test_fulfillment
+      param_attr = {
+        "payment_method": "upi",
+        "shipping": {
+          "waybill": "123456789",
+          "status": "rto",
```

Comment:
please add test case for failure scenario

Reply:
Added 

https://github.com/razorpay/razorpay-ruby/blob/1fcd3fa00ae1048e83a282d329e3f3d0bdf6b1cb/test/razorpay/test_order.rb#L102